### ISSUE TITLE
Support relative paths in pyproject.toml's [tool.esbonio.sphinx.pythonCommand]

### ISSUE THREAD

**Issue by neongreen-sc** [swyddfa/esbonio#914]
### What would you like to see

Right now it seems like paths have to be absolute; would be great if I could specify a path relative to the pyproject.toml file (to make life easier now that VSCode python env is no longer inherited)

**Comment by neongreen-sc** [comment]
(although given that the paths are different between Windows and Linux anyway.. I suppose it doesn't help _that_ much, compared to smth like "let me just give a path to .venv"? which is harder to implement)

**Comment by alcarney** [comment]
I agree that the experience with `esbonio.sphinx.pythonCommand` and venvs is not ideal at the moment. I have been thinking about adding a config syntax like `${venv:/path/to/env}` which would expand to `/path/to/env/bin/python` or `path/to/env/Scripts/python.exe` depending on the current OS.

As for the relative path, perhaps it's a simple as extending #847 to also cover `esbonio.sphinx.pythonCommand`? :thinking:
